xrally_kubernetes/tasks/validators.py

```diff
@@ -28,3 +28,31 @@ def validate(self, context, config, plugin_cls, plugin_cfg):
         if "kubernetes" not in context["platforms"]:
             self.fail("There is no specification for kubernetes platform in "
                       "selected environment.")
+
+
+@validation.configure(name="map_keys")
+class MapKeysParameterValidator(validation.Validator):
+    """Check that parameter contains specified keys.
+
+    :param param_name: Name of parameter to validate
+    :param keys: List of keys accepted
+    :param missed: Allow to accept optional parameter
```

Comment:
Let's use several parameters:
- required - list of required keys
- allowed - list of all allowed keys
- additional - whether additional keys are allowed. If list of allowed keys are specified, defaults to False, otherwise defaults to True.